.github/workflows/devcontainer-feature-test.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Test dev container feature"
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "ext/devcontainer/**"
```

Comment:
Did you see this comment?: https://github.com/radius-project/radius/pull/6930#issuecomment-1845850076

I'd like to put these files in `deploy/devcontainer-feature`

Reply:
Totally missed that one. Moved the files and synched the references from the workflows